Lesson-11/main.js

```diff
@@ -0,0 +1,146 @@
+const plusChildren = document.getElementById("plus-children");
+const minusChildren = document.getElementById("minus-children");
+const counterChildren = document.getElementById("counterChildren");
+
+const plusAdults = document.getElementById("plus-adults");
+const minusAdults = document.getElementById("minus-adults");
+const counterAdult = document.getElementById("counterAdult");
+
+const placeholderChildren = document.getElementById("placeholderChildren");
+const placeholderRoom = document.getElementById("placeholderRoom");
+
+const plusRooms = document.getElementById("plus-rooms");
+const RoomsMinus = document.getElementById("RoomsMinus");
+const counterRooms = document.getElementById("counterRooms");
+
+const copySelect = document.getElementById("copySelect");
+const filter = document.getElementById("age_select");
+const filterForm = document.getElementById("filterForm");
+
+const adultsButton = document.getElementById("adultsButton");
+const childrenButton = document.getElementById("childrenButton");
+const roomButton = document.getElementById("roomButton");
+
+plusAdults.addEventListener("click", () => {
+  counterAdult.innerHTML++ === 30
+    ? (counterAdult.innerHTML = 30)
+    : counterAdult.innerHTML;
+});
+
+plusAdults.addEventListener("click", () => {
+  placeholderCounter.innerHTML++ === 30
+    ? (placeholderCounter.innerHTML = 30)
+    : placeholderCounter.innerHTML;
+});
+
+minusAdults.addEventListener("click", () => {
+  placeholderCounter.innerHTML-- === 0
+    ? (placeholderCounter.innerHTML = 0)
+    : placeholderCounter.innerHTML;
+});
+
+minusAdults.addEventListener("click", () => {
+  counterAdult.innerHTML-- === 0
+    ? (counterAdult.innerHTML = 0)
+    : counterAdult.innerHTML;
+});
+
+//2
+plusChildren.addEventListener("click", () => {
+  counterChildren.innerHTML++ === 10
+    ? (counterChildren.innerHTML = 10)
+    : counterChildren.innerHTML;
+
+  filter.style.display = filter.style.display === "block" ? "block" : "block";
+
+  let p_prime = copySelect.cloneNode;
+
+  counterChildren.innerHTML < 0
+    ? (p_prime = copySelect.cloneNode(false))
+    : (p_prime = copySelect.cloneNode(true));
+
+  if (counterChildren.innerHTML == 10) {
+    p_prime = copySelect;
+  }
+
+  filterForm.appendChild(p_prime);
+});
+
+plusChildren.addEventListener("click", () => {
+  placeholderChildren.innerHTML++ === 10
+    ? (placeholderChildren.innerHTML = 10)
+    : placeholderChildren.innerHTML;
+});
+
+minusChildren.addEventListener("click", () => {
+  placeholderChildren.innerHTML-- == 0
+    ? (placeholderChildren.innerHTML = 0)
+    : placeholderChildren.innerHTML;
+});
+
+let fil = document.getElementById("pro1");
```

Comment:
I really recomment you install eslint! Also bad naming of IDs
![image](https://user-images.githubusercontent.com/20189313/141701587-3d98e52e-371d-41da-8da3-5a2c5fa1358c.png)